src/commands/apphosting-secrets-grantaccess.ts

```diff
@@ -73,7 +73,7 @@ export const command = new Command("apphosting:secrets:grantaccess <secretNames>
 
     const accounts = secrets.toMulti(secrets.serviceAccountsForBackend(projectNumber, backend));
 
-    await Promise.all(
+    await Promise.allSettled(
```

Comment:
Wanna explicitly handle errors here?

Or we could just go back to using `Promise.all()` instead... 🤷🏼 

e.g.
```
const results = await Promise.allSettled(
  secretList.map((secretName) => 
    secrets.grantSecretAccess(projectId, projectNumber, secretName, accounts)
  )
);

const failures = results.filter(result => result.status === 'rejected');
if (failures.length > 0) {
  throw new Error(`Failed to grant access to ${failures.length}/${results.length} secrets`);
 }
```